Geometry/HGCalCommonData/plugins/dd4hep/DDHGCalTBModule.cc

```diff
@@ -62,21 +62,21 @@ static long algorithm(dd4hep::Detector& /* description */, cms::DDParsingContext
     edm::LogVerbatim("HGCalGeom") << "Layer [" << i << "] with material type " << layerType[i] << " sensitive class "
                                   << layerSense[i];
 #endif
-  const auto& zMinBlock = args.value<double>("zMinBlock");  // Starting z-value of the block
-  const auto& rMaxFine = args.value<double>("rMaxFine");    // Maximum r-value for fine wafer
-  const auto& waferW = args.value<double>("waferW");        // Width of the wafer
-  const auto& waferGap = args.value<double>("waferGap");    // Gap between 2 wafers
-  const auto& absorbW = args.value<double>("absorberW");    // Width of the absorber
-  const auto& absorbH = args.value<double>("absorberH");    // Height of the absorber
+  auto zMinBlock = args.value<double>("zMinBlock");  // Starting z-value of the block
+  auto rMaxFine = args.value<double>("rMaxFine");    // Maximum r-value for fine wafer
+  auto waferW = args.value<double>("waferW");        // Width of the wafer
+  auto waferGap = args.value<double>("waferGap");    // Gap between 2 wafers
+  auto absorbW = args.value<double>("absorberW");    // Width of the absorber
+  auto absorbH = args.value<double>("absorberH");    // Height of the absorber
 #ifdef EDM_ML_DEBUG
   edm::LogVerbatim("HGCalGeom") << "DDHGCalTBModule: zStart " << zMinBlock << " rFineCoarse " << rMaxFine
                                 << " wafer width " << waferW << " gap among wafers " << waferGap << " absorber width "
                                 << absorbW << " absorber height " << absorbH;
 #endif
-  const auto& slopeB = args.value<std::vector<double> >("SlopeBottom");   // Slope at the lower R
-  const auto& slopeT = args.value<std::vector<double> >("SlopeTop");      // Slopes at the larger R
-  const auto& zFront = args.value<std::vector<double> >("ZFront");        // Starting Z values for the slopes
-  const auto& rMaxFront = args.value<std::vector<double> >("RMaxFront");  // Corresponding rMax's
+  const auto& slopeB = args.value<std::vector<double> >("SlopeBottom");  // Slope at the lower R
+  const auto& slopeT = args.value<std::vector<double> >("SlopeTop");     // Slopes at the larger R
+  auto zFront = args.value<std::vector<double> >("ZFront");              // Starting Z values for the slopes
```

Comment:
why are these now copies rather than const refs? It doesn't seem like the values are changed anywhere later.

Reply:
put back